aixplain/modules/model.py

```diff
@@ -249,3 +252,36 @@ def run_async(self, data: Union[Text, Dict], name: Text = "model_process", param
             if resp is not None:
                 response["error"] = msg
         return response
+    
+    def check_finetune_status(self):
+        """Check the status of the FineTune model.
+
+        Raises:
+            Exception: If the 'TEAM_API_KEY' is not provided.
+
+        Returns:
+            str: The status of the FineTune model.
+        """
+        if self.api_key == "":
+            raise Exception(
+                "A 'TEAM_API_KEY' is required to run a model. For help, please refer to the documentation (https://github.com/aixplain/aixplain#api-key-setup)"
+            )
```

Comment:
Lucas, here's one good contribution that you can make to the SDK. Please create a python decorator for checking the TEAM_API_KEY presence. Annotate all the functions that need the API KEY presence with that decorator.

Treat this comment as optional for this PR.

Reply:
@lucas-aixplain can you please remove this line?